Program.cs

```diff
@@ -48,6 +45,12 @@
 catch (ArgumentException ex) {
 	Console.Error.WriteLine(ex.Message);
 }
+catch (SignerNotFoundException) {
+	Console.WriteLine("[Error]: Unable to ad-hoc sign macos executable on this system.");
+	Console.WriteLine("         \"codesign\" from the Xcode command line tools, or Quill(https://github.com/anchore/quill)");
+	Console.WriteLine("         is required on PATH to patch macOS binaries.");
```

Comment:
Could reduce coupling if this was the exception message, and we just print `ex.Message`, but doesn't really matter